src/replication.c

```diff
@@ -12,6 +12,7 @@
 #include <errno.h>
 #include <sys/prctl.h>
 #include <sys/eventfd.h>
+#include </usr/include/assert.h>
```

Comment:
having relative path is better

Reply:
Since no assert is included at this point of time, I removed inclusion of file as per your comments.